ddtrace/contrib/pytest/_efd_utils.py

```diff
@@ -71,18 +80,18 @@ def efd_handle_retries(test_id, item, when, original_result, test_outcome):
     item.ihook.pytest_runtest_logreport(report=final_report)
 
 
-def efd_get_failed_reports(terminalreporter):
+def efd_get_failed_reports(terminalreporter: _pytest.terminal.TerminalReporter) -> t.List[_pytest.reports.TestReport]:
     return terminalreporter.getreports(_EFD_RETRY_OUTCOMES.EFD_ATTEMPT_FAILED)
 
 
-def _efd_handle_retries(item, original_outcome: _TestOutcome, force_teardown: bool = False) -> EFDTestStatus:
+def _efd_handle_retries(item) -> EFDTestStatus:
```

Comment:
Can we add type hinting here?
```suggestion
def _efd_handle_retries(item: pytest.Item) -> EFDTestStatus:
```